_events/sr2017/southampton-tech-day-december.md

```diff
@@ -0,0 +1,29 @@
+---
+title: Southampton December Tech Day 
+date: 2016-12-17 09:00:00
+layout: event
+type: tech day
```

Comment:
Is there something we can do to make it easier to get this sort of thing right? Perhaps a list of the proper types in the README or something the CI can do to validate a list? (Or both, though we'd want to ensure that there was only actually one canonical list and everything else referenced that).